layouts/_default/profile.html

```diff
@@ -19,147 +19,83 @@
 
 {{ define "main" }}
 <div class="container">
-  <div class="row">
-    <h1 class="col col-12">{{.Params.blurb}}</h1>
-  </div>
-  <div class="printer-hidden my-4 print-page">
-    <a href="#!" onclick="window.print()">{{ i18n "print" }}</a>
-  </div>
+    <h1 class="mb-xs">{{.Params.blurb}}</h1>
+    <img src="/okf/verein/team-2025.jpeg" style="width: 100%"/>
 </div>
+
 <div class="container">
-  <div class="printer-hidden">
-    <div class="col col-12 col-sm-10 offset-sm-1 col-md-10 offset-md-1 col-lg-8 offset-lg-2">
-      <div class="row c__additional no-gutters">
-        <div class="col-12 col-md-6">
-          <figure class="license">
-            <img src="/okf/verein/verein_intro_3.jpg" alt="">
-          </figure>
-        </div>
-        <div class="col-12 col-md-6">
-          <div class="c__additional__info">
-            <p>{{ .Params.box_title }}</p>
+    <div class="l__profile">
+        <h2>Wir arbeiten für ein offenes digitales Gemeinwohl </h2>
+        <p>Die Open Knowledge Foundation Deutschland (OKF) ist ein gemeinnütziger Verein mit Sitz in Berlin. Wir stärken
+            Zivilgesellschaft, Demokratie und Teilhabe im digitalen Zeitalter. Wir entwickeln Tools, organisieren
+            Bildungsangebote, pflegen Communities, initiieren Kampagnen und bringen unsere Expertise in politische
+            Prozesse ein. Unsere Projekte fördern Transparenz, Offenheit und technologische Souveränität.
+        </p>
+    </div>
+</div>
+
+<section class="l__content__bg--grey l__breakout my-d">
```

Comment:
take out section with "Politische Forderungen für offenes Wissen" here, this needs to be integrated somewhere else or in a different way.